### PR TITLE
Docs: Add docatl command examples as alternatives to curl

### PR DESCRIPTION
**Pull request by reglim**
fixes: #312

### CODE REVIEW

File: doc/getting-started.md [hunk 1]
```diff
@@ -36,6 +29,12 @@ For example to upload the file `docs.zip` as version `1.0.0` for `awesome-projec
 curl -X POST -F "file=@docs.zip" http://localhost:8000/api/awesome-project/1.0.0
 ```
 
+The same thing using `docatl`:
```
Comment (fliiiix) [inline doc/getting-started.md:32]: I would change that to just: `Using docalt:`

File: doc/getting-started.md [hunk 2]
```diff
@@ -6,14 +6,7 @@ The most convenient way to interact with docat is with it's official CLI tool, [
...
 
-After you've installed `docatl`, just point it to your documentation folder, which will be packaged and pushed to `docat`.
-To do this, use the following command:
-
-```sh
-docatl push --host http://localhost:8000 ./docs/ awesome-project 1.0.0
-```
-
-Use `docatl --help` to discover all commands available to manage your `docat` documentation!
+The `docatl` alternatives to the `curl` commands are outlined below. Do note that the host address and api-key can be omitted if specified in a `.docatl.yml` file. See the `docatl` documentation for more information.
```
Comment (fliiiix) [inline doc/getting-started.md:9]: I would still keep that here 
```
Use `docatl --help` to discover all commands available to manage your `docat` documentation!
```

> See the `docatl` documentation for more information.

should contain a link to some docs

> The `docatl` alternatives to the `curl` commands are outlined below.

not very happy with this sentence but failed to come up with a better structure

**Review by fliiiix — CHANGES_REQUESTED** [review]

File: doc/getting-started.md [hunk 3]
```diff
@@ -63,6 +70,10 @@ Each Project can be claimed **exactly once**, so best store the token safely.
 ```sh
 curl -X GET http://localhost:8000/api/awesome-project/claim
 ```
+Using `docatl`:
```
Comment (randombenj) [inline doc/getting-started.md:73]: Please use consistent space before ` ```sh `

Comment (reglim) [reply doc/getting-started.md:73]: You mean just one space everywhere?

Comment (reglim) [reply doc/getting-started.md:74]: Fixed :)

**Pull request merged**